src/pages/ProfilePage.tsx

```diff
@@ -21,154 +20,136 @@ import PressableWithoutFocus from '@components/Pressable/PressableWithoutFocus';
 import ScreenWrapper from '@components/ScreenWrapper';
 import Text from '@components/Text';
 import UserDetailsTooltip from '@components/UserDetailsTooltip';
-import withLocalize, {withLocalizePropTypes} from '@components/withLocalize';
+import useLocalize from '@hooks/useLocalize';
 import useThemeStyles from '@hooks/useThemeStyles';
-import compose from '@libs/compose';
 import Navigation from '@libs/Navigation/Navigation';
 import * as PersonalDetailsUtils from '@libs/PersonalDetailsUtils';
 import {parsePhoneNumber} from '@libs/PhoneNumber';
 import * as ReportUtils from '@libs/ReportUtils';
 import * as UserUtils from '@libs/UserUtils';
 import * as ValidationUtils from '@libs/ValidationUtils';
+import type {ProfileNavigatorParamList} from '@navigation/types';
 import variables from '@styles/variables';
-import * as PersonalDetails from '@userActions/PersonalDetails';
-import * as Report from '@userActions/Report';
-import * as Session from '@userActions/Session';
+import * as PersonalDetailsActions from '@userActions/PersonalDetails';
+import * as ReportActions from '@userActions/Report';
+import * as SessionActions from '@userActions/Session';
```

Comment:
Let's turn this imports back as they was, and update OnyxTypes import to be 
`import type * as OnyxTypes from '@src/types/onyx';`
This way we can use them like `OnyxTypes.PersonalDetails`, `OnyxTypes.Report`, etc

Reply:
I think it is better to import  only the needed types

Reply:
@pasyukevich I agree with @VickyStash comment in this case, by doing your way you had to change the other imports to avoid conflicts, we don't need that if we use `OnyxTypes.*` pattern